### PR TITLE
Update sickbeard.py for PEPs 8, 263 and streamline datetime call

### PR DESCRIPTION
**Pull request by labrys**
##### Code changes:
- Streamline `datetime` call used to time-stamp backup files

##### Formatting changes:
- PEP 0008 - Add and remove blank lines for consistency
- PEP 0263 - Add encoding declaration
- Add comments for pylint codes

### CODE REVIEW

**Comment by SiCKRAGEB0T** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your report has all the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure you send a PR against the DEVELOP branch
- [x] Don't send big changes in one go. Split up big PRs into multiple smaller (easier manageble) PRs

Please make sure you also read [contribution guide](https://github.com/SiCKRAGETV/SickRage/blob/master/contributing.md) and followed all the steps.

Thanks!

File: SickBeard.py [hunk 1]
```diff
@@ -20,6 +21,7 @@
 # Check needed software dependencies to nudge users to fix their setup
 
 # pylint: disable=W0703
+#                 Catching too general exception
```
Comment (miigotu) [inline SickBeard.py]: =P Several of these

Also, many of thse can be disabled in your pylintrc. I have been fine tuning mine to soon add it to the repo

**Comment by labrys** [comment]
I know, but when I see a check disabled in the code, I like to know why it's there without having to look it up.  I figured it wouldn't hurt to add comments as I come across them.

**Comment by miigotu** [comment]
Still, ones like "too many lines in module", "too many instance attributes", "line too long" etc are crazyness

Also need `ignored-modules=sickbeard.logger`

**Comment by labrys** [comment]
Agreed

**Comment by labrys** [comment]
And just for clarification, I didn't add the pylint disables to the code, they were already present.  I just added the comment about what the existing codes referenced.

**Comment by miigotu** [comment]
AHAHAHA silly tired me.

Still though, badly formatted comment =P:
`#                 Catching too general exception`

**Comment by labrys** [comment]
Hmm, that's odd, I have my tab key mapped to insert spaces... I'll fix that here in a few.

**Comment by labrys** [comment]
Ah, found it.  Setting changed after updating PyCharm.

**Comment by labrys** [comment]
Fixed comments and rebased onto latest dev branch with #3090

**Pull request merged**